index.html

```diff
@@ -0,0 +1,221 @@
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">Campaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <h2 class="hero-med-title">"Hello! Rugby fans"</h2>
+        <h1 class="hero-main-title">
+          Welcome to the home of Cape Town Rugby 10's
+        </h1>
+        <p class="hero-subtitle">
+          A fun festival where people from all around the world come together to
+          support their teams.
+        </p>
+        <h3 class="venue">3 – 4 FEBRUARY 2023 | ENTRIES OPEN 1 APRIL 2022</h3>
+      </div>
+    </section>
+    <!-- program section -->
+    <section class="section-program" id="program">
+      <header class="program-header">
+        <h2 class="program-header-title">Main Program</h2>
+        <div class="program-header-title-underline underline"></div>
+      </header>
+      <div class="program-cards">
+        <div class="card1 card">
+          <i class="fa fa-desktop screen-icon"></i>
+          <h3 class="lecture-titel work-title">Lecture</h3>
+          <div class="info">
+            <p class="lecture-subtitle">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt,
+              temporibus.
+            </p>
+          </div>
+        </div>
+        <div class="card2 card">
+          <i class="fa-solid fa-coins"></i>
+          <h3 class="exhibition-titel work-title">Exhibition</h3>
+          <div class="info">
+            <p class="lecture-subtitle">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt,
+              temporibus.
+            </p>
+          </div>
+        </div>
+        <div class="card3 card">
+          <i class="fa-solid fa-comment-dollar"></i>
+          <h3 class="forum-titel work-title">forum</h3>
+          <div class="info">
+            <p class="lecture-subtitle">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt,
+              temporibus.
+            </p>
+          </div>
+        </div>
+        <div class="card4 card">
+          <i class="fa-solid fa-chalkboard-user"></i>
+          <h3 class="workshop-titel work-title">workshop</h3>
+          <div class="info">
+            <p class="lecture-subtitle">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt,
+              temporibus.
+            </p>
+          </div>
+        </div>
+        <div class="card5 card">
+          <i class="fa-solid fa-link"></i>
+          <h3 class="party-titel work-title">party</h3>
+          <div class="info">
+            <p class="lecture-subtitle">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt,
+              temporibus.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button class="program-btn btn">Join Cape Town 10's Rugby</button>
```

Comment:
- [ ] Kindly ensure that your button design matches the design guideline specified 👍🏽 

| Design Guideline | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190245710-7ee8f08e-2f0c-4539-b075-2196f0e95df8.png)  | ![image](https://user-images.githubusercontent.com/63473584/190245776-27d528db-11f0-46c8-bc29-faa39e975ae7.png)  |